src/main/resources/org/jenkinsci/plugins/workflow/steps/EnvStep/help.html

```diff
@@ -1,6 +1,7 @@
 <div>
 Sets one or more environment variables within a block.
-These are available to any external processes spawned within that scope.
+The names of environment variables are case insensitive but case preserving, i.e., setting `Foo` will change the value of `FOO` if it already exists.
```

Comment:
```suggestion
The names of environment variables are case insensitive but case preserving, that is, setting `Foo` will change the value of `FOO` if it already exists.
```

Reply:
Yes, but case-insensitive and case-preserving should be hyphenated. Thanks!